### PR TITLE
Add postgresql support

### PR DESCRIPTION
**Pull request by Opencontent**

### CODE REVIEW

File: classes/sqliimportitem.php [hunk 1]
```diff
@@ -163,8 +163,17 @@ public function __get( $name )
...
+        {
+            if ( eZINI::instance()->variable( 'DatabaseSettings', 'DatabaseImplementation' ) == 'ezpostgresql' )
+            {
+                $safeSerialized = str_replace( "~~NULL_BYTE~~", "\0", $this->attribute( 'options_serialized' ) );
+                $this->options = unserialize( $safeSerialized );
+            }
+            else
+            {
+                $this->options = unserialize( $this->attribute( 'options_serialized' ) );
+            }
```
Comment (lolautruche) [inline classes/sqliimportitem.php]: As this block is repeated many times, maybe it's worth factor it somewhere? Like in SQLIImportUtils?

**Comment by lolautruche** [comment]
Sorry for being so long to reply! It's been a while I didn't check my Github notification :blush: 

Thank you so much for this pull-request! I'll merge it as soon as you take my comment into account :smiley:

**Comment by Opencontent** [comment]
@lolautruche  I moved the repeated code in SQLIImportUtils as you suggested
thank you for this wonderful extension :)

**Comment by lolautruche** [comment]
Merged!

Thanks :)

**Comment by lolautruche** [comment]
Just released v1.3.0 with your patch

**Pull request merged**